lib/primitives/input.js

```diff
@@ -169,10 +169,11 @@ class Input extends bio.Struct {
    * for JSON serialization.
    * @param {Network} network
    * @param {Coin} coin
+   * @param {Path} path
    * @returns {Object}
    */
 
-  getJSON(network, coin) {
+  getJSON(network, coin, path) {
```

Comment:
Looking for additional places this is invoked besides in `lib/primitives/tx.js` and I don't see any that could use `path` being passed along.

It does get invoked in the `txdb` but I don't think that `path` is relevant here.
https://github.com/handshake-org/hsd/blob/325747e25a4843c374e37b96fbd161797052953b/lib/wallet/txdb.js#L3499

Reply:
Agreed. Thanks for the check.